mindmap/mindmap.py

```diff
@@ -281,12 +249,11 @@ def upload_file(self, data, _suffix="") -> None:
             user = self.get_current_user()
             data["path_prefix"] = self.anonymous_user_id(user)
```

Comment:
Change this `path_prefix` for something more scoped like 'mindmaps_uploads/{{anonymous_user_id}}' or something similar

Reply:
Yes, the full path is obtained from the `get_file_key` method.